tests/src/JIT/config/extra/project.json

```diff
@@ -5,6 +5,7 @@
     "System.Runtime.Extensions": "4.0.10-beta-*",
     "System.Collections": "4.0.10-beta-*",
     "System.Runtime.InteropServices": "4.0.21-beta-*",
+    "System.Runtime.InteropServices.RuntimeInformation": "4.0.0-beta-*",
```

Comment:
Nit: if you want to get ahead of the game on #3396, you could pin this to a particular version (e.g. 23516, which is the version dnx chose as indicted by the lock file).